File: pset6/mario.py
import cs50

while True:
    print("Height: ", end="")
    Height = cs50.get_int()
    if Height > 0 and Height < 23:
        break
    
n = Height

for i in range(n):
    print(" "*(Height - 1), end="")
    print("#"*(2+i), end="")
    print(" ", end="")
    print("#"*(2+i))
    Height = Height-1
    
    

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //ask for the Height of the piramid in the interval
    int Height;
    do
    {
        printf("Height: ");
        Height = GetInt();
    }
    while(Height < 0 || Height > 23);
    
    //Generate the piramid
    int n = Height;
    for(int i = 0; i < n; i++)
    {
        for(int spaces = (Height - 1); spaces > 0; spaces--)
        {
            printf(" ");
        }
        for(int hashes = (n + 2); hashes > Height; hashes--)
        {
            printf("#");
        }
        printf("\n");
        Height--;
    }
}